sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveUtilsSuite.scala

```diff
@@ -62,6 +67,39 @@ class HiveUtilsSuite extends QueryTest with SQLTestUtils with TestHiveSingleton
     }
   }
 
+  test("IsolatedClientLoader shall use system classpath first") {
```

Comment:
Shall we add a JIRA prefix in the title of the test?

Reply:
Sure, let me put that